packages/ng/forms/rich-text-input/plugins/tag/tag-node.ts

```diff
@@ -194,8 +198,8 @@ function domConversionFunction(domNode: Node): DOMConversion {
 	};
 }
 
-export function $createTagNode(key = '', description?: string, partial = false): TagNode {
-	return new TagNode(key, description, partial);
+export function $createTagNode(key = '', description?: string, disabled = false): TagNode {
+	return new TagNode(key, description, disabled);
```

Comment:
disabled n'est plus utilisé dans ce cas

```suggestion
export function $createTagNode(key = '', description?: string): TagNode {
	return new TagNode(key, description, false);
```